libs/dal/src/repositories/notification-template/notification-template.entity.ts

```diff
@@ -121,13 +123,3 @@ export class StepFilter {
 
   children: FilterParts[];
 }
-
-export interface ITriggerVariable {
-  name: string;
-  type: TemplateVariableTypeEnum;
-}
-
-export interface ITriggerReservedVariable {
-  type: TriggerContextTypeEnum;
-  variables: ITriggerVariable[];
-}
```

Comment:
Do we have a reason for having those in the DAL?
i noticed that they are duplicated and not aligned with the interface in the Shared lib.